app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
-<h1>Here is the detail info for a given user</h1>
+<div class="user-container">
+    <div class="user-wrapper">
+        <%= image_tag(@user.photo)%>
+        <div class="user-details">
+            <h2><%= @user.name %></h2>
+            <h5>Number of posts: <%= @user.posts_counter %></h5>
+        </div>
+    </div>
+    <div class="bio-wrapper">
+        <h3>bio:</h3>
+        <p><%= @user.bio %></p>
+    </div>
+    <div class= 'post-container'>
+        <% @post.each_with_index do |post,index| %>
+            <div class="post-wrapper">
+                    <h2>Post #<%= index+1 %></h2>
+                    <p><%= post.text %></p>
+                    <h5>Comments: <%= post.comments_counter %> , Likes: <%= post.likes_counter %></h5>
+            </div>
+        <% end %>
+    </div>
```

Comment:
- [ ] Please use the `#recent_posts` method to render the user's 3 most recent posts. You are currently rendering all the user's posts in the user show page.
![image](https://github.com/addisalemseifu/my_blog_app/assets/46839548/624fcd3b-ed51-40f3-8642-13ea504960e9)

Reply:
well resolved ✔️ 

![image](https://github.com/addisalemseifu/my_blog_app/assets/38088818/0accff3b-765b-40fc-afee-aec087334d61)

[OPTIONAL] It would be best to make each post clickable on the user show page too, currently I'm unable to navigate to the post show page unless I go to the post index page.